### DISCUSSION TITLE
How can I run a check on a schedule without materializing its asset?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18051] [Q&A] [answered]
can you use a schedule without a target job? say if we wanted to run a check daily at a given time

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700082658161419?thread_ts=1700082658.161419&cid=C01U954MEER).*

**Comment by johannkm** [comment, marked as answer]
You can define a job that targets asset checks and schedule that job. Docs: https://docs.dagster.io/concepts/assets/asset-checks#via-sensors-and-schedules
